R/auth.R

```diff
@@ -10,43 +10,27 @@
 #' please consider [registering for an API
 #' key](https://api.delphi.cmu.edu/epidata/admin/registration_form).
 #'
-#' API keys are strings and can be set in two ways. If the environment variable
-#' `DELPHI_EPIDATA_KEY` is set, it will be used automatically. Environment
-#' variables can be set either in the shell or by editing your `.Renviron` file,
-#' which will ensure the setting applies to all R sessions. See `?Startup` for a
-#' description of `Renviron` files and where they can be placed.
-#'
-#' Alternately, the API key can be set from within a session by using
-#' `set_api_key()`, which sets the R option `delphi.epidata.key` using
-#' `options()`. If this option is set, it is used in preference to the
-#' environment variable, so you may change keys within an R session. R options
-#' are not preserved between sessions, so `set_api_key()` must be run every time
-#' you open R. Alternately, you can have R set the option at startup by adding
-#' it to your `.Rprofile`; see `?Startup` for a description of `Rprofile` files
-#' and where they can be placed.
+#' API keys are strings read from the environment variable `DELPHI_EPIDATA_KEY`.
+#' We recommend setting your key with `save_api_key()`, which will modify your
+#' `.Renviron` file and will persist your key across R sessions (see `?Startup`
+#' for a description of `.Renviron` files). Alternatively, you can modify the
+#' environment variable in the shell or with `Sys.setenv()`, but these will not
+#' persist across sessions.
```

Comment:
```suggestion
#' We recommend setting your key with `save_api_key()`, which will modify an
#' applicable `.Renviron` file, which will be read in automatically when you
#' start future R sessions (see [`?Startup`][base::Startup] for details on
#' `.Renviron` files). Alternatively, you can modify the environment variable at
#' the command line before/while launching R, or inside an R session with
#' [`Sys.setenv()`], but these will not persist across sessions.
```

Reply:
GitHub's just randomly not highlighting changes for me in some places.  This reworded "your", "persist your key across", "for a description of", and "in the shell", plus added links to `?Startup` and `Sys.setenv()`.

Reply:
Btw, `?Startup` and `Sys.setenv()` is auto-linked by roxygen (see [here](https://cmu-delphi.github.io/epidatr/dev/reference/get_api_key.html) for example).